src/App.jsx

```diff
@@ -1,3 +1,21 @@
+import React from 'react'
+import { UserProvider } from './context/UserContext'
+import { Home } from './views/Home/Home'
+import { Layout } from './views/Layout/Layout'
+ 
+
 export default function App() {
-  return <h1>Go Away</h1>;
+  return (
```

Comment:
Nice work here wrapping your provider your provider on the entire app to hold your state